documentation/MIGRATION guide for bumping up to v30.0.0.md

```diff
@@ -0,0 +1,196 @@
+## Guide for bumping Javascript/Typescript SDK to version 30.0.0+
+
+This document is intended to help users bump Javascript/Typescript SDK version to version 30.0.0+ as there are some significant changes since version 30.0.0.
```

Comment:
we should not refer to track 2 as versions 30.0.0+ but more like next-generation JavaScript SDK.

We can add a section such as "How can I get the next-generation libraries", then we say something like "Versions above 30.0.0 for certain namespaces are next generation SDKs, we can also provide the example link to a Track 2 package"

Reply:
some of the new services will use track 2 default and they will not be version 30.0.0+

Reply:
change it into the next-generation.